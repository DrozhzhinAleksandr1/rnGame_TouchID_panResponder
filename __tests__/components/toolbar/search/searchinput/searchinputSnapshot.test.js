import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { SearchInput } from '/components/toolbar/search/searchinput/searchinput';

it('create snapshot Search', () => {
    const renderer = new ShallowRenderer()
    let SearchSnapshot = renderer.render(<SearchInput />);
    expect(SearchSnapshot).toMatchSnapshot();
});