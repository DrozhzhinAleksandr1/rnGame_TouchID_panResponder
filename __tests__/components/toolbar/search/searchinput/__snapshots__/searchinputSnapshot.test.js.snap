// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot Search 1`] = `
<Component
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <TouchableWithoutFeedback
    accessibilityLabel="searchFocus_mainID"
    onPress={[Function]}
    testID="searchFocus_mainID"
  >
    <Component
      style={
        Object {
          "alignItems": "center",
          "height": "100%",
          "justifyContent": "center",
          "width": 56,
        }
      }
    >
      <Icon_search_gray />
    </Component>
  </TouchableWithoutFeedback>
  <TextInput
    accessibilityLabel="searchInput_TitleID"
    allowFontScaling={true}
    autoCorrect={false}
    autoFocus={true}
    onChangeText={[Function]}
    selectionColor="#5A6772"
    spellCheck={false}
    style={
      Object {
        "color": "#FFF",
        "flex": 1,
        "fontFamily": "Roboto-Regular",
        "fontSize": 18,
      }
    }
    testID="searchInput_TitleID"
    underlineColorAndroid="transparent"
  />
  <TouchableWithoutFeedback
    accessibilityLabel="search_closeID"
    onPress={[Function]}
    testID="search_closeID"
  >
    <Component
      style={
        Object {
          "alignItems": "center",
          "height": "100%",
          "justifyContent": "center",
          "width": 56,
        }
      }
    >
      <Icon_close />
    </Component>
  </TouchableWithoutFeedback>
</Component>
`;
